contracts/astroport-liquidity-helper/src/msg.rs

```diff
@@ -13,11 +12,6 @@ pub type ExecuteMsg = GenericExecuteMsg<CallbackMsg>;
 
 #[cw_serde]
 pub enum CallbackMsg {
-    ProvideLiquidity {
-        assets: AssetList,
-        min_out: Uint128,
-        pool: AstroportPool,
-    },
```

Comment:
how come we remove this? why is it not needed anymore?

Reply:
It's not needed anymore since we now don't do a simulate swap in cw-dex pool.provide_liquidity

Reply:
See the comment on the handler that was removed:
```
/// CallbackMsg handler to provide liquidity with the given assets. This needs
/// to be a callback, rather than doing in the first ExecuteMsg, because
/// pool.provide_liquidity does a simulation with current reserves, and we do a
/// swap in the top level ExecuteMsg, which means the reserves would be wrong in
/// the provide liquidity simulation.
```

Reply:
ahh i see. min_out is passed in the BalancingProvideLiquidity message